cmd/controller/controllers/nodeconfigscrapper/spec/aks.go

```diff
@@ -0,0 +1,125 @@
+package spec
+
+import (
+	"github.com/samber/lo"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func AKS(nodeName, jobName string) *batchv1.Job {
+	//https://raw.githubusercontent.com/aquasecurity/kube-bench/v0.6.9/job-aks.yaml
+	return &batchv1.Job{
+		TypeMeta: metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: jobName,
+			Annotations: map[string]string{
+				"autoscaling.cast.ai/disposable": "true",
+			},
+			Labels: map[string]string{
+				"app":                          "kube-bench",
+				"app.kubernetes.io/managed-by": "castai",
+			},
+		},
+		Spec: batchv1.JobSpec{
+			BackoffLimit: lo.ToPtr(int32(0)),
+			Template: corev1.PodTemplateSpec{
+				Spec: corev1.PodSpec{
+					HostPID:                      true,
+					NodeName:                     nodeName,
+					RestartPolicy:                "Never",
+					AutomountServiceAccountToken: lo.ToPtr(true),
+					Containers: []corev1.Container{
+						{
+							Name:  "kube-bench",
+							Image: "<placeholder>",
+							SecurityContext: &corev1.SecurityContext{
+								ReadOnlyRootFilesystem:   lo.ToPtr(false),
+								AllowPrivilegeEscalation: lo.ToPtr(false),
+							},
+							Resources: corev1.ResourceRequirements{
+								Requests: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse(requestCPU),
+									corev1.ResourceMemory: resource.MustParse(requestMem),
+								},
+								Limits: corev1.ResourceList{
+									corev1.ResourceCPU:    resource.MustParse(limitCPU),
+									corev1.ResourceMemory: resource.MustParse(limitMem),
+								},
+							},
+							Command: []string{
+								"/usr/local/bin/kvisor-linter",
+							},
+							Args: []string{
+								"node-config-scrapper",
+							},
+							Env: []corev1.EnvVar{
+								{
+									Name:  "NODE_NAME",
+									Value: nodeName,
+								},
+							},
+							VolumeMounts: []corev1.VolumeMount{
```

Comment:
Do we need these mounts? Since `HostPID=true` we should be able to read the files doing `/proc/1/root/...`